src/main/webapp/assets/script.js

```diff
@@ -71,6 +72,8 @@ function onLoad() {
   posts = fetchFakePosts();
   collegeLocation = fetchFakeCollegeLocation();
 
+  addPosts();
```

Comment:
also, completely up to you but if you want to have posts that are the same as the fake markers on the map, you could use the `posts` object on line 72 that contains fake results of a fake GET request! this might also make it easier in the future to integrate addPosts with the real GET results

Reply:
Ahh great idea! Just implemented it.